package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/a11y-tests-web",
-  "version": "6.0.1",
+  "version": "6.1.0",
```

```suggestion
  "version": "6.0.1",
```

Please put this back - I'll change the version here and in package-lock.json as part of the release process.